lib/utils/core-flags.js

```diff
@@ -0,0 +1,14 @@
+const { core } = require('./cli-flags');
+
+const names = core
+    .map(({ alias, name }) => {
+        if (name === 'help') return [];
+        if (alias) {
+            return [`--${name}`, `-${alias}`];
+        }
+        return [`--${name}`];
+    })
+    .reduce((arr, val) => arr.concat(val), []);
+
+module.exports.names = names;
+module.exports.info = core;
```

Comment:
```js
module.exports = {names, core};
```

Reply:
I prefer current ones as I want to keep `info` and variable name instead of `core`.

Reply:
```

module.exports = {
    names,
    info:core
}

```

Reply:
Cool. It's night here, will update tommorow 👍

Reply:
We're all from the same time zone :sweat_smile: